libs/zzz/stats/src/mappedStats/wengine/maps/BigCylinder.ts

```diff
@@ -0,0 +1,20 @@
+import type { WengineKey } from '@genshin-optimizer/zzz/consts'
+import { allStats } from '../../../allStats'
+
+const key: WengineKey = 'BigCylinder'
+const data_gen = allStats.wengine[key]
```

Comment:
There is a getparam function in stas/wengine.ts that does the translation already